modules/installation-osp-accessing-api-floating.adoc

```diff
@@ -39,10 +43,28 @@ api.<cluster_name>.<base_domain>.  IN  A  <API_FIP>
 +
 [NOTE]
 ====
-If you do not control the DNS server you can add the record to your `/etc/hosts` file instead. This action makes the API accessible to you only, which is not suitable for production deployment but does allow installation for development and testing.
+If you do not control the DNS server, you can add the record to your `/etc/hosts` file. This action makes the API accessible to you only, which is not suitable for production deployment, but does allow installation for development and testing.
 ====
 
```

Comment:
PR https://github.com/openshift/installer/pull/4219/ changed the behavior for UPI (and _only_ for UPI): it's now necessary to create the bootstrap FIP manually and pass it via the `os_bootstrap_fip` variable in the inventory.

Reply:
@mandre And how does this work for you? https://github.com/openshift/openshift-docs/pull/25216/commits/eba8ab85dd42e4ada33ce9b557d81bba993dec53